lib/pbench/server/database/models/datasets.py

```diff
@@ -637,7 +645,14 @@ class Metadata(Database.Base):
     #
     # NOTE: the entire "global" and "user" namespaces are writable, but only
     # specific keys in the "dataset" and "server" namespaces can be modified.
-    USER_UPDATEABLE_METADATA = [DATASET_NAME, DELETION, GLOBAL, USER]
+    USER_UPDATEABLE_METADATA = [
+        DATASET_NAME,
+        ARCHIVEONLY,
+        DELETION,
+        ORIGIN,
+        GLOBAL,
+        USER,
+    ]
```

Comment:
Ha; it was actually sorted by namespace, with the Pbench-owned `dataset` and `server` first, then `global`, and finally `user`. When I added the new ones I followed my convention with `DELETION` and didn't complicate the names with an unwieldy prefix, but also noted that it made them look unordered. Or, more unordered than they were.

Alphabetical has no meaning unless I rename the constants to match the values. Another possibility would be to put the "pure" namespaces (`GLOBAL` and `USER` first, followed by `DATASET_NAME` (why did *it* get a prefix?) and then the `SERVER` namespace keys in alphabetical order?

Reply:
I'm not wedded to alphabetical order:  my request is that it be obvious where in the list to add a new name or to look to see if an existing name is missing/etc.

Ordering by namespace is OK, but it's not entirely obvious which namespace each of these belongs to without looking at their values (I think I would prefer that the ordering be done based on the symbol name, so that referring to code elsewhere isn't required).  We could add an annotation (e.g., in a code comment) and then order based on that, if necessary.

But, this is a small thing which I would ignore unless you're making other changes (which...I presume you are... 🙂).